gunicorn/config.py

```diff
@@ -535,8 +535,11 @@ class Bind(Setting):
     desc = """\
         The socket to bind.
 
-        A string of the form: ``HOST``, ``HOST:PORT``, ``unix:PATH``. An IP is
-        a valid ``HOST``.
+        A string of the form: ``HOST``, ``HOST:PORT``, ``unix:PATH``,
+        ``fd://FD`. An IP is a valid ``HOST``.
+
+        .. versionchanged:: 19.10
```

Comment:
master is now targets 20.0, so I think this needs to be 20.0 instead of 19.10. We can discuss whether we should backport this to the 19.x branch later.